src/sortStudents.ts

```diff
@@ -1,16 +1,68 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name,
+  Surname,
+  Age,
+  Married,
+  AverageGrade,
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+function averageGradeFunc(grades: number[]): number {
+  let avarage: number = 0;
+  let sum: number = 0;
 
-export function sortStudents(students, sortBy, order) {
+  for (let i: number = 0; i < grades.length; i += 1) {
+    sum += grades[i];
+    avarage = sum / grades.length;
+  }
+
+  return avarage;
+}
+
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+): object[] {
   // write your function
+  let newArr: object[] = [];
+
+  if (sortBy === 0) {
```

Comment:
use switch/case and enum
```suggestion
  case SortType.Name:
```